src/Kunstmaan/TranslatorBundle/Command/ImportTranslationsFromFileCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Kunstmaan\TranslatorBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Exception\LogicException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class ImportTranslationsFromFileCommand
+ */
+final class ImportTranslationsFromFileCommand extends ContainerAwareCommand
```

Comment:
You are creating a new command. Maybe it's a good start to remove the container here already ?